make.sh

```diff
@@ -66,10 +76,13 @@ for kernel_version in "${kernel_versions[@]}"; do
 	if [ "${series}" = "4.14" ]; then
 		inc="$(find /usr/include -iregex '.+/asm/bitsperlong\.h$' | head -n 1)"
 		export CLANG="$clang '-I${inc%asm/bitsperlong.h}'"
+	elif [ "${kernel_version}" = "5.10.72" ]; then
```

Comment:
Let's avoid more corner cases here, especially the kind of change you've introduced below.

If a maintained kernel version doesn't build properly without modifications, we skip it until it's been fixed in that release line.